collector/src/main/java/com/navercorp/pinpoint/collector/receiver/grpc/GrpcReceiver.java

```diff
@@ -159,15 +160,37 @@ private void addService() {
         }
     }
 
+    private void shutdownServer() {
+        if (server == null || server.isTerminated()) {
+            return;
+        }
+
+        final long maxWaitTime = serverOption.getGrpcMaxTermWaitTimeMillis();
+
+        server.shutdown();
+        if (awaitServerTermination(maxWaitTime)) {
+            return;
+        }
+
+        server.shutdownNow();
+        awaitServerTermination(1000);
+    }
+
+    private boolean awaitServerTermination(long maxWaitTimeMillis) {
+        try {
+            return server.awaitTermination(maxWaitTimeMillis, TimeUnit.MILLISECONDS);
+        } catch (InterruptedException ignored) {}
```

Comment:
```
catch (InterruptedException e) {
            Thread.currentThread().interrupt();
}
```
I think it would be better to fix it to call `Thread.interrupt()`.